sprint-implementation/02-project-prerequisites.md

```diff
@@ -0,0 +1,16 @@
+From outline:
+
+### Project prerequisites
```

Comment:
This is not sprint or agile specific. But it is important, so I'd leave it here until we have a better place to put it.